test/e2e/client_tls_test.go

```diff
@@ -905,7 +892,7 @@ func TestMTLSWithCRLs(t *testing.T) {
 			}
 			t.Cleanup(func() { assertDeleted(t, kclient, &clientCAConfigmap) })
 			icName := types.NamespacedName{
-				Name:      "mtls-with-crls",
+				Name:      names.SimpleNameGenerator.GenerateName("mtls-with-crls"),
```

Comment:
Slightly conspicuous that this prefix doesn't have a trailing "-"; is that intentional?

Reply:
Fixed